one_fm/api/utils.py

```diff
@@ -11,25 +11,29 @@ def response(code, title, msg, data=None):
 
 def get_reports_to_employee_name(employee):
```

Comment:
I have a similar method in https://github.com/ONE-F-M/One-FM/blob/5d3da8da3bc07004e522e2107ccfc1cf01feeda4/one_fm/api/tasks.py#L463. I believe we should unify these methods.

Reply:
@SalShai your method returns user_Id this one returns employee name(pk)

They are two different method